app/models/user_content.rb

```diff
@@ -2,6 +2,10 @@ class UserContent < ApplicationRecord
   belongs_to :user
   belongs_to :content
   belongs_to :content_list
+<<<<<<< HEAD
```

Comment:
This change shall be reverted of fixed. Originates from an unresolved merge conflict.